docs/changelog.md

```diff
@@ -68,6 +68,9 @@ permalink: /changelog/
 
 * **support-ktx**
   * Added `Collection.crossProduct` to retrieve the cartesian product of two `Collections`.
+  
+* **service-glean**
+  * ⚠️ **This is a breaking change**: `Glean.enableTestingMode` is now `internal`. Tests can use the `GleanTestRule` to enable testing mode.
```

Comment:
Maybe we could put the documentation for this in `mozilla/glean`, merge that first and then link to it here?

Reply:
Good idea! As soon as I have all the reviews here, I'll start tinkering with the glean-core one, land the docs there, link them here and then merge this :)